data/ir-ads

```diff
@@ -0,0 +1,355 @@
+# List of ads and tracking company in Iran.
+# Thanks to PersianBlocker project
+# https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerHosts.txt
+
+#-------------------------v Persian Ads/Trackers/Annoyances Filters v-----------------------#
+# >> A <<
+analyt.ir
+
+scripts-ads.s3.ir-thr-at1.arvanstorage.com
+error-tracking.arvancloud.com
+
+adexo.ir
+adskav.com
+advn.ir
+
+amaroid.net
+
+analytics.aasaam.com
+analytics-2.aasaam.com
+analytics-3.aasaam.com
+
+adwised.com
+adwisedfs.com
+adwised.azureedge.net
+# TODO: Remove adwised.azureedge.net, but navatv.com and baran-music.com still links to it
+
+adivery.com
+adnegah.net
+adpulse.ir
+adsima.net
+adtrace.io
+affili.ir
+amarfa.ir
+ayyaar.ir
+# ^ Used by irna.ir
+
+adro.co
+affstat.digikala.com
+affiliate.digikala.com
+
+# >> B <<
+behinava.com
+radar.bayan.ir
+backority.ir
+
+# >> C <<
+cayot.ir
+# Example: tamasha.com/v/DDlMB
+
+chavosh.org
+
+clickaval.com
+clickyab.com
+chabok.io
+
+congoro.com
+congoro.ir
+
+# >> D <<
+24d.ir
+
+denutility.com
+davedbux.ir
+
+deemanetwork.com
+deema.agency
+deemaagency.ir
+
+# https://forum.adguard.com/index.php?threads/vigiato-net-resolved.44366/
+# https://github.com/AdguardTeam/AdguardFilters/issues/79983
+click.digiato.com
+# ^ Used by Digiato, Vigiato, Rooziato
+
+# >> E <<
+
+# >> F <<
+facepop.org
+farsbux.ir
+fastclick.ir
+
+farakav.com
+#  TODO: Check for breakages (used by tamasha.com), investigate into more tracking
```

Comment:
If in doubt, please do not introduce it.

Reply:
Removed.